twarc2sql/file_utils/object_to_table.py

```diff
@@ -148,12 +184,13 @@ def tweet_object_to_table(
     """
     # validate_object(tweet_object, "tweet_object")
     tweet_object["tweet_type"] = 0
-    refrenced_tweet_column_processing(tweet_object, tables)
+    referenced_tweet_column_processing(tweet_object, tables)
 
     tweet_object = expand_dict_column(tweet_object, "public_metrics")
     tweet_object = expand_dict_column(tweet_object, "edit_controls")
 
```

Comment:
Remove this to fix bug 
```suggestion
    tweet_object = expand_dict_column(tweet_object, "edit_controls")

```[#24](https://github.com/unna97/twarc2sql/issues/24)
```suggestion
    tweet_object = expand_dict_column(tweet_object, "edit_controls")

```

Reply:
Remove this to fix bug 
```suggestion
    tweet_object = expand_dict_column(tweet_object, "edit_controls")

```[#24](https://github.com/unna97/twarc2sql/issues/24)
```suggestion
    tweet_object = expand_dict_column(tweet_object, "edit_controls")

```